sdk/search/azure-search/src/main/java/com/azure/search/models/IndexerExecutionInfo.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/**
+ * Represents the current status and execution history of an indexer.
+ */
+@Fluent
+public final class IndexerExecutionInfo {
```

Comment:
Missing 'name'

Reply:
@careyjmac I don't see a `name` property in the docs or the Swagger. Are you maybe thinking of `ItemError`?

Reply:
AzureSearch\Source\Search\Product\SearchCore\Models\2019-05-06\IndexerExecutionInfo.cs line 11 has a name property in the backend.  That's what I was cross referencing with.  Its just the indexer name though so maybe it doesn't need to be exposed here since when retrieving this object it should be pretty obvious to the user which indexer it correlates with.

Reply:
@Careyjmac It's unfortunate that a redundant property is now part of the API surface, since you can't remove it now without breaking backward compatibility. Please ensure that the property is documented [here](https://docs.microsoft.com/rest/api/searchservice/get-indexer-status) and add it to the relevant Swaggers. We won't block the PR on this.